packages/synapse-react-client/src/components/EntityHeaderTable/EntityHeaderTable.tsx

```diff
@@ -0,0 +1,542 @@
+                          background: '#fff',
+                        }}
+                      >
+                        {header.isPlaceholder ? null : (
+                          <>
+                            <div
+                              {...{
+                                className: header.column.getCanSort()
+                                  ? 'SRC-hand-cursor'
+                                  : '',
+                                onClick:
+                                  header.column.getToggleSortingHandler(),
+                              }}
+                            >
+                              {flexRender(
+                                header.column.columnDef.header,
+                                header.getContext(),
+                              )}
+                              {{
+                                asc: (
+                                  <IconSvg
+                                    icon={'sortUp'}
+                                    wrap={false}
+                                    sx={{
+                                      color: 'primary.main',
+                                      backgroundColor: 'none',
+                                      float: 'right',
+                                      marginRight: '5px',
+                                    }}
+                                  />
+                                ),
+                                desc: (
+                                  <IconSvg
+                                    icon={'sortDown'}
+                                    wrap={false}
+                                    sx={{
+                                      color: 'primary.main',
+                                      backgroundColor: 'none',
+                                      float: 'right',
+                                      marginRight: '5px',
+                                    }}
+                                  />
+                                ),
+                              }[header.column.getIsSorted() as string] ?? null}
+                            </div>
+                            {header.column.getCanFilter() &&
+                            totalRowCount > UNMANAGEABLE_SUBJECT_COUNT ? (
+                              <div>
+                                <Filter column={header.column} table={table} />
+                              </div>
+                            ) : null}
+                          </>
+                        )}
+                        {header.column.getCanResize() && (
+                          <div
+                            className={`resizer ${
+                              header.column.getIsResizing() ? 'isResizing' : ''
+                            }`}
+                            onMouseDown={header.getResizeHandler()}
+                            onTouchStart={header.getResizeHandler()}
+                          />
+                        )}
```

Comment:
Does this work with the hard-coded percentages for column width?

Reply:
I don't think so